manifest.yml

```diff
@@ -33,7 +33,7 @@ applications:
   services:
     - c2-dev-elasticsearch
     - c2-dev-db
-    - c2-dev-ups-s3
+    - c2-dev-s3
```

Comment:
I changed this to match what was already existing in the cloud.gov/development env.